response_operations_ui/templates/message_select_survey.html

```diff
@@ -0,0 +1,49 @@
+{% extends "layouts/base.html" %}
+{% block page_title %}Messages | Survey Data Collection{% endblock %}
+
+{% block main %}
+</br>
+
+<div role="main" id="main" class="page__main">
+
+  <div class="grid">
+    <div class="grid__col col-30@l">
+      <h1 class="jupiter">Filter messages by survey</h1>
+        <form method="POST">
+          {% if survey_list %}
+            <div class="field field--radio field--multiplechoice">
+              <fieldset>
+                <legend class="field__legend mars u-mb-s">Please select a survey to see messages for your team</legend>
+                <div class="field field--radio">
+                  {% for survey in survey_list %}
+                  <div class="field__item js-focusable-box">
+                      <input class="input input--radio js-focusable" name="radio-answer" value={{ survey }} id={{ survey }} type="radio">
+                      <label class="label label--inline venus " for={{ survey }}>{{ survey }}</label>
+                  </div><br />
+                  {% endfor %}
+                  <div class="field__item js-focusable-box">
+                      <input class="input input--radio js-focusable" name="radio-answer" value="sand-and-gravel" id="sand-and-gravel" type="radio">
+                      <label class="label label--inline venus " for="sand-and-gravel">Sand & Gravel</label>
+                  </div><br />
+                  <div class="field__item js-focusable-box">
```

Comment:
I couldn't find a way to get the ID correctly, it would come up as 'sand' because it's stopping at the space in 'sand & gravel'... mostly for when we add the acceptance tests, but sand as an ID could be fine?